web/src/main/java/se/inera/intyg/webcert/web/service/intyg/IntygServiceImpl.java

```diff
@@ -632,53 +641,85 @@ public boolean isRevoked(String intygsId, String intygsTyp, boolean coherentJour
 
     @Override
     public List<IntygWithNotificationsResponse> listCertificatesForCareWithQA(IntygWithNotificationsRequest request) {
-        List<Utkast> utkastList;
-        if (request.shouldUseEnhetId()) {
-            utkastList = utkastRepository.findDraftsByPatientAndEnhetAndStatus(
-                DaoUtil.formatPnrForPersistence(request.getPersonnummer()), request.getEnhetId(), Arrays.asList(UtkastStatus.values()),
-                moduleRegistry.listAllModules().stream().map(IntygModule::getId).collect(Collectors.toSet()));
-        } else {
-            utkastList = utkastRepository.findDraftsByPatientAndVardgivareAndStatus(
-                DaoUtil.formatPnrForPersistence(request.getPersonnummer()), request.getVardgivarId(),
-                Arrays.asList(UtkastStatus.values()),
-                moduleRegistry.listAllModules().stream().map(IntygModule::getId).collect(Collectors.toSet()));
-        }
+        final var res = new ArrayList<IntygWithNotificationsResponse>();
```

Comment:
Ändra till ett tydligare variabelnamn

Reply:
Fixat. Checkar in ändringar samtidigt när du är klar med reviewn.